source/usage-examples/updateMany.txt

```diff
@@ -3,3 +3,59 @@ Update Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can update multiple documents in a collection by using the ``UpdateMany()``
+method.
+
+The following example passes a query filter and an update parameter to the
+``UpdateMany()`` method. The query filter matches documents in the
+``listingsAndReviews`` collection if the value of their "address.market"
```

Comment:
```suggestion
``UpdateMany()`` method, which matches documents in the
``listingsAndReviews`` collection with the value of their ``address.market``
```
Suggestion: Keeping the wording consistent with your UpdateOne page, and field names should be monospaced.

Reply:
accepted, but I think the word "if" should stay in place of "with". Does the sentence seem too long now?

Reply:
A bit. In the past, we converted sentences that seemed long to either a few shorter sentences or a bulleted list of steps/action.